src/App.tsx

```diff
@@ -15,71 +15,74 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+interface State {
+  selectedGood: string | null;
+}
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    selectedGood: 'Jam',
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  handleClear = () => this.setState({ selectedGood: '' });
```

Comment:
The `handleClear` function should set the `selectedGood` to `null` instead of an empty string (`''`).

or you need to change  type in the interface State to
  `selectedGood: string;`